EpsilonAuxBmsV2/Core/Src/Tasks/ContactorFunctions/CheckCurrent.c

```diff
@@ -0,0 +1,23 @@
+#include "CheckCurrent.h"
+
+/*
+Reads the current and determines if it's low or not based on the number of contactors closed.
+*/
+uint8_t isCurrentLow(uint8_t numContactorsClosed)
```

Comment:
The reason why we did that was because of the setup time of turning on the `CURRENT_SENSE` pin. However, we decided to just have it on all the time because we don't really care about ultra low power. So there's no reason to do the checking to see if the value is settled